sortableidentifier/src/test/java/com/github/awsjavakit/identifiers/SortableIdentifierTest.java

```diff
@@ -0,0 +1,36 @@
+package com.github.awsjavakit.identifiers;
+
+import static com.github.awsjavakit.testingutils.RandomDataGenerator.randomString;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+class SortableIdentifierTest {
+
+  public static final int ELIMINATE_PROBABILITY_OF_ACCIDENTAL_SUCCESS = 5;
+
+  @Test
+  void shouldBeUniversallyUnique() {
+    var s1 = SortableIdentifier.create();
+    var uuid = s1.getUuid();
+
+    assertThat(uuid.toString()).isSubstringOf(s1.toString());
+  }
+
+  @Test
+  void shouldBeSortableBasedOnWhenItWasCreated() throws InterruptedException {
```

Comment:
Could you achieve the same effect with `@RepeatedTest(5)`?

Reply:
thanks